include/mbedtls/config.h

```diff
@@ -56,6 +56,22 @@
 #define MBEDTLS_HAVE_ASM
 
 /**
+ * \def MBEDTLS_NO_INT64_DIVISION
+ *
+ * The platform lacks support for 64-bit division.
+ *
+ * Used in:
+ *      include/mbedtls/bignum.h
+ *      library/bignum.c
+ *
+ * Uncomment to prevent the use of 64-bit division, even if a 64-bit type
+ * is available. This allows building for a platform that lacks a hardware
+ * 64-bit division, if linking with a software implementation of division
+ * is not desired. Note that 32-bit division is required.
+ */
+//#define MBEDTLS_NO_INT64_DIVISION
```

Comment:
Yes, that's a good idea. At the moment we define `MBEDTLS_HAVE_UDBL` if it looks like a double-width type is available. The compilation option would disable this detection.